Makefile

```diff
@@ -30,7 +30,8 @@ all-modules:
 	@echo $(ALL_MODULES) | tr ' ' '\n' | sort
 
 .PHONY: all
-all: common gotest otelcontribcol otelcontribcol-unstable
+#all: common gotest otelcontribcol otelcontribcol-unstable
```

Comment:
Any reason for this change ?

Reply:
Right now on _macos_ gotest fails w/ linker error therefore as temporary remedy - it should not be part of changes.